internal/core/command/get_test.go

```diff
@@ -61,9 +64,11 @@ var testCommand = contract.Command{
 
 func TestNewGetCommandWithCorrelationId(t *testing.T) {
 	expectedCorrelationIDHeaderValue := "Testing"
+	req := newRequestForGetWithContentType(clients.ContentTypeCBOR)
 	testContext := context.WithValue(context.Background(), clients.CorrelationHeader, expectedCorrelationIDHeaderValue)
-	getCommand, _ := NewGetCommand(testDevice, testCommand, "", testContext, nil, logger.NewMockClient())
+	getCommand, _ := NewGetCommand(testDevice, testCommand, testContext, nil, logger.NewMockClient(), req)
 	actualCorrelationIDHeaderValue := getCommand.(serviceCommand).Request.Header.Get(clients.CorrelationHeader)
+
 	if actualCorrelationIDHeaderValue == "" {
 		t.Errorf("The populated GetCommand's request should contain a correlation ID header value")
 	}
```

Comment:
This looks like it is duplicated in the test table for `TestNewGetCommandContentType`. If so can we remove this test?